README.md

```diff
@@ -1,7 +1,13 @@
-# open-ce
+<p align="center">
+  <img src="https://avatars0.githubusercontent.com/u/68873540?s=400&u=a02dc4156e50cdffb23172aba7133e44381885d4&v=4" alt="Open-CE Logo" width="30%">
+</p>
 
 ![Builder Unit Tests](https://github.com/open-ce/open-ce/workflows/Open-CE%20Builder%20Unit%20Tests/badge.svg)
 [![Builder Unit Test Coverage](https://codecov.io/gh/open-ce/open-ce/branch/master/graph/badge.svg)](https://codecov.io/gh/open-ce/open-ce)
+![Python version](https://img.shields.io/badge/python-3.6%20%7C%203.7-blue.svg)
```

Comment:
Should be able to add 3.8 here as well. The build scripts work with 3.8.

Reply:
I was a little torn here. The build tools support 3.8, but the packages only support up to 3.7, right?

Reply:
They should build with 3.8 as well.